hawc/apps/assessment/views.py

```diff
@@ -362,6 +367,13 @@ class DatasetCreate(BaseCreate):
 class DatasetRead(BaseDetail):
     model = models.Dataset
 
+    def get_object(self, **kwargs):
+        obj = super().get_object(**kwargs)
+        if obj.user_can_view(self.request.user):
```

Comment:
this is nitpicky, but it's nice to have bailout conditions instead of these conditions. Propose changing this and the api to something like this:

```python
if not obj.user_can_view(self.request.user):
    raise PermissionDenied()
return obj
```

https://www.itamarweiss.com/personal/2018/02/28/return-early-pattern.html